index.html

```diff
@@ -1,11 +1,24 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Hypnosis</title>
 </head>
 <body>
-    <h1> hola mundo </h1>
+  <header>
+    <h1><a href="#">Hypnosis&trade;</a></h1>
+    <nav class="navbar" role="menubar">
+      <ul>
+        <li><a href="#" tabindex="1">Home</a></li>
+        <li><a href="#" tabindex="2"><strike>World</strike></a></li>
+        <li><a href="#" tabindex="3"><strike>Polities</strike></a></li>
+        <li><a href="#" tabindex="4">Tech</a></li>
+        <li><a href="#" tabindex="5">Sports</a></li>
+        <li><a href="#" tabindex="6">Culture</a></li>
```

Comment:
Good job!!! tabindex attr makes elements focusable, but the `<a>` is enough for that :blush: